src/VirtualMonoRepo/VirtualMonoRepo.Tasks.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Consider placing all of the tasks files in a Tasks subfolder.  IMO this helps make the code more approachable.

Reply:
Should I do the same for the SB tasks?

Reply:
That's your call.  I see it as a dead end code path that parts will be deleted or moved into VMR so any changes don't provide long term value.